src/test_l1_tvSettings.c

```diff
@@ -13288,7 +13271,7 @@ void test_l1_tvSettings_positive_GetCurrentComponentHue (void)
 		UT_ASSERT_AUTO_TERM_TRUE( (Hue >=0 && Hue <= 100));
 	}
 
-	/* Step 04: Calling tvsettings termination and expecting the API to return success */
+	/* Step 05: Calling tvsettings termination and expecting the API to return success */
 	result = TvTerm();
```

Comment:
similarly add for SetCMSState(true) for SetCurrentComponentLuma

Reply:
Moved to SetCurrentComponentLuma